### PR TITLE
Add location resource updates

### PR DESCRIPTION
**Pull request by napster235**
Description
----
Add location resource updates

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

**Comment by napster235** [comment]
Before
Screenshot 2022-09-02 at 11 06 27
After commit
Screenshot 2022-09-02 at 11 06 07
 ede7077

**Review by MariaPislaru1 — COMMENTED** [review]
@napster235 FYI

**Review by lstefan4 — APPROVED** [review]

File: content/millennium/r4/base/entities/location.md [hunk 1]
```diff
@@ -11,15 +11,24 @@ title: Location | R4 API
...
 
+* The following [HL7® FHIR® US Core Implementation Guide STU 4.0.0](https://hl7.org/fhir/us/core/STU4/){:target="_blank"} Profiles are supported by this resource:
+
+  * [US Core Location Profile](http://hl7.org/fhir/us/core/STU4/StructureDefinition-us-core-location.html){:target="_blank"}
+
 The following fields are returned if valued:
 
 * [Location id](http://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
 * [Status](http://hl7.org/fhir/R4/location-definitions.html#Location.status){:target="_blank"}
+  * [Location Status](http://hl7.org/fhir/R4/valueset-location-status.html){:target="_blank"} ([Active](http://hl7.org/fhir/R4/codesystem-location-status.html#location-status-active){:target="_blank"} \| [Inactive](http://hl7.org/fhir/R4/codesystem-location-status.html#location-status-inactive){:target="_blank"})
```
Comment (cerbs) [inline content/millennium/r4/base/entities/location.md:22]: The Location Status, Mode, and Telecom changes should be documented in the Terminology Bindings section and not in the supported fields list.

To do that, add the appropriate details [here](https://github.com/cerner/fhir.cerner.com/blob/db9f6bd5ad3c94738a19df8c1c2bc0ce98c897c1/lib/resources/r4/location.yaml) and they will be pulled into the existing Terminology Bindings table

**Review by cerbs — CHANGES_REQUESTED** [review]

File: content/millennium/r4/base/entities/location.md (hunk 1 above)
Comment (napster235) [reply content/millennium/r4/base/entities/location.md]: Solved in [ce62e40](https://github.com/cerner/fhir.cerner.com/pull/887/commits/ce62e40f81fa6174d4a9f7b8d9b45d7089d3d3f7)

**Comment by napster235** [comment]
After commit [ce62e40](https://github.com/cerner/fhir.cerner.com/pull/887/commits/ce62e40f81fa6174d4a9f7b8d9b45d7089d3d3f7)
Screenshot 2022-09-07 at 13 24 19

**Review by davischristi — APPROVED** [review]

**Review by cerbs — APPROVED** [review]

**Pull request merged**